Makefile

```diff
@@ -46,8 +47,8 @@ build-contract-tests:
 	@mkdir -p test-service/_checkouts
 	@rm -f $(CURDIR)/test-service/_checkouts/ldclient
 	@ln -sf $(CURDIR)/ $(CURDIR)/test-service/_checkouts/ldclient
-	@if [ "$(OTP_VER)" = "26.x" ]; then\
-		echo Dialyze for OTP 26;\
+	@if [ "$(ERL_VERSION)" -ge "26" ]; then\
```

Comment:
The dialyzer settings changes in 26 and the new settings don't work earlier. This changes the 26 check to be a 26+ check.